### ISSUE TITLE
Questions about training S-CNN

### ISSUE THREAD

**Issue by jinwchoi** [zhengshou/scnn#2]
Hi Zheng,

I have some questions about training S-CNN. 

1) In your S-CNN paper, you are using windows of size 16,32,64,...,512 with 16 frames.  How did you generate C3D volume mean files for different window sizes? I know the number of frames are same across window sizes. But you need to adjust the "sampling rate" when you are computing volume means. Then there will be 6 different volume mean files. How do you select a volume mean file from the 6 files for training S-CNN?  There is only one volume mean file in the github repository: "train01_16_128_171_mean.binaryproto".

2) How many positive/negative examples are sampled from each video given a window size? Can you share your "input data file" please? It would be really helpful.

Thank you!

**Comment by zhengshou** [comment]
Hi Jinwoo,

Thank you for your interest in our work.

1) We use the original mean files of C3D pre-trained model on sports1m instead of generating ours.

2) In our paper section 3.2, we have discussed about the strategy used to generate positive/negative examples and assign label for training in detail. If there is anything unclear to you, feel free to let me know.

Best,
Zheng

**Comment by carloscaetano** [comment]
Sorry for re-opening it but... @jinwchoi , did you get the input files? If so,  can you share it? Thanks in advance.

**Comment by buaa-luzhi** [comment]
Hi，@jinwchoi 
I am sorry to trouble you!
For peoposal networks, I was in trouble during the production of the training set. I don't know how to determine the candidate segment label. Is 1 or 0 for candidate segments label?
I just want to know how to determine training set labels for candidate segments in untrimmed videos.
Do you have any script implementation?

Thank you in advance!
